cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec *rspec.Spec) error {
+	fmt.Println("validating readonlyPaths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
On Thu, May 26, 2016 at 07:30:15PM -0700, Ma Shimiao wrote:
> It seems not to remount readonlyPaths and just change their
> permisson.

I think the spec wording leaves the implemenation ambiguous.  And
attempting a read clearly covers all possible implementations, while
checking the mode only covers a single implementation.